components/dashicon/index.js

```diff
@@ -829,7 +856,6 @@ export default class Dashicon extends wp.element.Component {
 			<svg
 				aria-hidden
 				role="img"
-				focusable="false"
```

Comment:
I don't know what implications it has, but if you are confident doing it then 👍

Reply:
Argh, comparing https://github.com/WordPress/gutenberg/blob/master/components/dashicon/index.js with the upstream https://github.com/WordPress/dashicons/blob/master/react/index.jsx reveals we have pushed commits to Gutenberg that weren't pushed back upstream. 

Pressing pause in this PR as I sync that stuff up.